webpack/JobWizard/steps/HostsAndInputs/buildHostQuery.js

```diff
@@ -4,7 +4,7 @@ export const buildHostQuery = (selected, search) => {
   const hostCollectionsSearch = `(host_collection_id ^ (${hostCollections
```

Comment:
If the [test data](https://github.com/theforeman/foreman_remote_execution/blob/master/webpack/JobWizard/__tests__/fixtures.js#L140) is to be trusted then for host collections searching by id is actually correct. My only machine with katello is currently out of order, so it will take me a while before I can check it